.lintstagedrc.json

```diff
@@ -1,3 +1,6 @@
 {
-  "*.{ts,tsx}": ["npm run lint"]
+  "*.{ts,tsx}": [
+    "npm run lint",
+    "prettier --write"
```

Comment:
It seems `prettier --write` doesn't run `prettier` as expected:

![lint-staged](https://github.com/neurobagel/react-query-tool/assets/77515879/a8fc1cec-c5c6-403f-90a5-c465d0cfccce)

I suggest using `npm run format` instead.

Reply:
Thanks for letting me know. While reading the [documentation](https://btholt.github.io/complete-intro-to-react-v5/eslint-prettier#npmyarn-scripts), I also noticed that `npm run format` should be used for this.